forcetree.c

```diff
@@ -501,11 +584,45 @@ force_get_prev_node(int no)
         return 0;
     }
 }
+
+/*Sets the node softening on a node.*/
+static void
+force_set_node_softening(struct NODE * pnode, const int new_type, const double hsml)
+{
+    if(pnode->f.MaxSofteningType == 7)
```

Comment:
I don't quite get why we need MaxSofteningType. If you recall anything, could you add a comment in the struct definition before you forget?

Reply:
Maybe we don't - let me think about it a bit.

Reply:
I think the Softening flags usage is wrong in an important respect, and it is causing the problems I have been having with my neutrino simulations. I will add a comment to this branch and file a PR for the other.